examples/neural_networks/plot_mlp_alpha.py

```diff
@@ -36,7 +36,8 @@
 
 classifiers = []
 for i in alphas:
-    classifiers.append(MLPClassifier(solver='lbfgs', alpha=i, random_state=1,
+    classifiers.append(MLPClassifier(solver='lbfgs', alpha=i,
```

Comment:
Uhm hold-on before merging. We should check if normalizing the data might have an effect on the time to converge as well

Reply:
oh yes we have a standard-scaler but this is really a bad practise to show `fit_transform` on the full `X`

Reply:
```suggestion
    	    classifiers.append(make_pipeline(	
	        StandardScaler(),	
	        MLPClassifier(solver='lbfgs', alpha=i, random_state=1,	
	                      hidden_layer_sizes=[100, 100], max_iter=2000)	
	    ))
```

Reply:
we should import `from sklearn.pipeline import make_pipeline` on the top of the file

Reply:
Right, let's go with:
```python
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=.4)
scaler = StandardScaler()
X_train = scaler.fit_transform(X_train)
X_test = scaler.transform(X_test)
```

Reply:
for me it seems faster. @TiphaineV could you check locally it change on your benchmark>

Reply:
> Right, let's go with:

No, no. Let's use a proper pipeline since we don't introspect anything in the network :_)